.eslintrc.cjs

```diff
@@ -1,16 +1,26 @@
+/* eslint-env node */
 module.exports = {
   env: {
     browser: true,
     es2021: true,
   },
   extends: [
-    'airbnb-base',
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
   ],
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint',
+    "no-loops",
+    "no-comments"],
+  root: true,
   parserOptions: {
     ecmaVersion: 12,
     sourceType: 'module',
   },
   rules: {
-    "import/extensions":0
+    'import/extensions': 0,
+    "no-console": 1,
+    "no-loops/no-loops": 2,
```

Comment:
Try removing loops else you will face error of no-loop, if you can't avoid removing loops change the value of this property to 1